### DISCUSSION TITLE
ChromiumWebBrowser.EvaluateScriptAsync not available on c# library ?

### DISCUSSION THREAD

**Discussion by Merange** [cefsharp/CefSharp#4043] [General]
Hello,

I try to use WinForms.ChromiumWebBrowser.EvaluateScriptAsync within a UserForm in a library with .net Framework 4.7.2 on Visual Studio 2019.

I imported CefSharp with the Visual Studio 2019 integrated "Manage nuget packages" and the packages following packages were imported
>   package id="cef.redist.x64" version="99.2.9" targetFramework="net472" 
>   package id="cef.redist.x86" version="99.2.9" targetFramework="net472" 
>   package id="CefSharp.Common" version="99.2.90" targetFramework="net472" 
>   package id="CefSharp.WinForms" version="99.2.90" targetFramework="net472"

I put a code like this one in a method 
Browser.EvaluateScriptAsync(@"OnBtnClick ();");

Bowser was defined as CefSharp.WinForms.ChromiumWebBrowser by the WinForm designer

But there is a compile error indicating 

> Erreur	CS1061	'ChromiumWebBrowser' ne contient pas de définition pour 'EvaluateScriptAsync' et aucune méthode d'extension accessible 'EvaluateScriptAsync' acceptant un premier argument de type 'ChromiumWebBrowser' n'a été trouvée (une directive using ou une référence d'assembly est-elle manquante ?)	

What's happening ?

Thank you

**Comment by amaitland** [comment]
In future please use discussions to ask questions. 

You need to add a using statement for the CefSharp namespace. Visual Studio should have given you a quick fix suggestion to add the namespace. 

```c#
using CefSharp;
```

**Comment by Merange** [comment]
Thank you very much for your answer.
I was not used to add root namespace...
